.github/workflows/issues-loader.yaml

```diff
@@ -0,0 +1,45 @@
+name: issues-loader
+
+on:
+  schedule:
+    # Run job every hour
+    - cron: 0 */1 * * *
+  workflow_dispatch:
+    inputs:
+      dryRun:
+        description: 'Flag to run the script without creating issues'
+        default: false
+        required: true
+        type: boolean
+
+permissions:
+  issues: write
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v4
+      - name: Set up Python 3.10
+        uses: actions/setup-python@v5
+        with:
+          python-version: "3.10"
+          cache: "pip"
+      - name: Install dependencies
+        run: pip install -r requirements.txt
+      - name: Run load issues
+        run: |
+          if [[ "${DRY_RUN}" == "true" ]]; then
+            echo "Running load issues in dry run mode"
+            python load_issues.py --dry-run
+          else
+            echo "Running load issues"
+            python load_issues.py
+          fi
+        env:
+          # if the workflow was triggered by a workflow_dispatch event,
+          # use the value of the dryRun input parameter; otherwise, use 'false'
+          DRY_RUN: ${{ github.event_name == 'workflow_dispatch' && inputs.dryRun || 'false' }}
```

Comment:
why the `|| 'false'`?

Reply:
This case covers the regular case for the cron, which **shouldn't** do a "dry run". Since if the event is not a workflow dispatch, then the left side of `||` becomes false regardless of the value of `inputs.dryRun`. I think this logic works...